index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.10.2/font/bootstrap-icons.css"
+      integrity="sha384-b6lVK+yci+bfDmaY1u0zE8YYJt0TZxLEAFyYSLHId4xoVvsrQu3INevFKo+Xir8e"
+      crossorigin="anonymous"
+    />
+    <title>Mobile-First</title>
+  </head>
+  <body>
+    <main>
+    <section class="parent">
+      <div>
+        <nav>
+          <!--container 1-->
+          <div>
+            <h5>My Logo</h5>
+          </div>
+          <!--container 2-->
+          <div>
+            <i class="bi bi-list"></i>
+          </div>
```

Comment:
- [ ] You can then wrap your hamburger button inside a `<nav>` tag for semantic reasons.

Reply:
As requested by the previous reviewer,  kindly wrap this menu item with a `nav` tag as mentioned earlier for semantics purposes. The HTML element represents a section of a page whose purpose is to provide a navigation link. 🤓